src/System.Management.Automation/engine/remoting/commands/PSRemotingCmdlet.cs

```diff
@@ -146,7 +146,7 @@ internal string GetMessage(string resourceString, params object[] args)
 
         #region Private Members
 
-        private static string s_LOCALHOST = "localhost";
+        private static readonly string s_LOCALHOST = "localhost";
```

Comment:
This should be `const string`

Reply:
These changes will be made in follow-up PR, see tracking issue #13914

Reply:
No, please make these changes here.  The Issue can be used to pick up the rest.